packer_update_centos_base.json

```diff
@@ -34,17 +34,13 @@
     {
       "type" : "shell",
       "expect_disconnect" : true,
-      "inline" : [
-        "sudo yum -y upgrade",
-        "echo Update Complete.  Rebooting.",
-        "sudo reboot ; sleep 30"
-      ]
+      "script" : "centos-upgrade-first-stage.sh"
```

Comment:
we need to restructure our folders for build scripts, just adding remark. will do it after release

Reply:
+1